aspnetcore/mvc/razor-pages/filter/sample/PageFilter/Pages/subFolder/About2.cshtml

```diff
@@ -0,0 +1,8 @@
+﻿@page
```

Comment:
Is this intentional? The folder name \ file name seem a bit suspect.

Reply:
I deleted about2. I don't need it to show:
```
services.AddMvc()
   .AddRazorPagesOptions(options =>
   {
       options.Conventions.AddFolderApplicationModelConvention(
           "/subFolder",
           model => model.Filters.Add(new SampleAsyncPageFilter(_logger)));
   });
```